sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/tests/WebJobsEventHubTestBase.cs

```diff
@@ -60,7 +62,7 @@ protected void ConfigureTestEventHub(IHostBuilder builder)
             });
         }
 
-        protected (JobHost, IHost) BuildHost<T>(Action<IHostBuilder> configurationDelegate = null, Action<IHost> preStartCallback = null)
+        protected (JobHost, IHost) BuildHost<T>(Action<IHostBuilder> configurationDelegate = null, Action<IHost> preStartCallback = null, InitialOffsetOptions initialOffsetOptions = null)
```

Comment:
You can use `configurationDelegate ` to set the `initialOffsetOptions` and avoid adding another parmeter.

Reply:
Should initialOffsetOptions  be removed?